QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
+QUEST_LV_0100_20150317_000053	$Mercenary Mirta
+QUEST_LV_0100_20150317_000054	{memo X}$Didn't I already warn you?{nl}Please be careful next time.
+QUEST_LV_0100_20150317_000055	$Historian Kepeck
+QUEST_LV_0100_20150317_000056	$Niels must have sent you.{nl}Well, if you are in a hurry, record it yourself and take it back to Niels.
+QUEST_LV_0100_20150317_000057	$When you go straight on the right road and cross the bridge, you will see the artifacts of the great King Zachariel.{nl}Record them and take the recordings to Niels. He will understand.
+QUEST_LV_0100_20150317_000058	$So this is Kepeck's record... Mercenary Mirta was looking for this.{nl}You helped confirm the safety of two individuals. Thank you for your efforts.
+QUEST_LV_0100_20150317_000059	$Ahylas Jonas
+QUEST_LV_0100_20150317_000060	{memo X}$Now take the unsealing stone and go to Broken Pillar Canyon. {nl}The Disciple of Gustas will teach you how to release the seal there.  there.
+QUEST_LV_0100_20150317_000061	{memo X}$Were you not guided yet?{nl}Take it to the Disciple of Gustas in Broken Pillar Canyon.{nl}It is made to let only the chosen one enter.
+QUEST_LV_0100_20150317_000062	$Disciple of Gustas
+QUEST_LV_0100_20150317_000063	$Have you unlocked the seal of Pagasa Cliff?{nl}It will start from now.
+QUEST_LV_0100_20150317_000064	$Words engraved into a rock
+QUEST_LV_0100_20150317_000065	$Do not touch or push the rock.
+QUEST_LV_0100_20150317_000066	$I was waiting for you.{nl}The next seal is located on the way to Collapsed Sanctum.
+QUEST_LV_0100_20150317_000067	$You don't know how eagerly we have waited for this moment.{nl}We really wanted to meet you.
+QUEST_LV_0100_20150317_000068	$Gustas Jonas
+QUEST_LV_0100_20150317_000069	$It's you.{nl}You are much more than what I had imagined.
+QUEST_LV_0100_20150317_000070	$Soldier Alan
+QUEST_LV_0100_20150317_000071	$You are a new soldier right?{nl}I received a patrolling order, but I guess you can go my behalf.
+QUEST_LV_0100_20150317_000072	$You're good for someone new.{nl}I had a horrible memory there once, so I don't want to go.
+QUEST_LV_0100_20150317_000073	$I'm remembering something...
+QUEST_LV_0100_20150317_000074	{memo X}$You know that we are dispatched here to defeat Gaigalas right?{nl}Rookie. Can you take this report to Commander Wallis?
+QUEST_LV_0100_20150317_000075	$Commander Wallis
+QUEST_LV_0100_20150317_000076	$Why did you bring this report without Alan coming?{nl}Are you a new soldier?
+QUEST_LV_0100_20150317_000077	$I can return home after we defeat Gaigalas.{nl}Well, I'd like to go home as soon as possible.
+QUEST_LV_0100_20150317_000078	$Wallis is at Bird Footprint Junction.
+QUEST_LV_0100_20150317_000079	$I don't care how good you are. You are new here.{nl}It is an order. Bring some sap from Kepa Raiders.
+QUEST_LV_0100_20150317_000080	$Soldiers these days, they lack guts.{nl}I hope you're a good influence to them.
```

Comment:
Ah, I missed that, thanks 

$Soldiers these days, they lack guts.{nl}I hope you'll be a good influence to them.